docs/user-guide/concepts.md

```diff
@@ -218,6 +221,61 @@ probably not a great idea.
 You can create a subnamespace from the command line via `kubectl hns create child
 -n parent`.
 
+<a name="hrq">
+
+### Hierarchical resource quotas (HRQs)
+
+When you want to give some amount of resources to `team-a`, and want them to be able to
+flexibly use resources in any of their subnamespaces, you create a `HierarchicalResourceQuota`
+in namespace `team-a`. The sum of all resources from all the subnamespaces of the
+members wont be over the amount of resources that is configured in
+`HierarchicalResourceQuota` of namespace `team-a`. All of the reasources of `team-a` are
+equally shared between the applications in their subnamespaces, which is very efficient.
+
+You have the struggle that each time some team calls you and asks you to modify their
+`ResourceQuota` or namespaces structure? HRQ fixes it!
+With the following teams tree:
+```
+company-a
+├── organization-a
+│   ├── org-a-team-1
+│   ├── org-a-team-2
+│   ...
+├── organization-b
+│   ├── org-b-team-1
+│   ├── org-b-team-2
+│   ...
+...
+```
+Instead of each team askin from the `cluster-admin` to modify their ResourceQuota, 
+you can make the following namespace hierarchy:
+```
+company-a-restrictions
```

Comment:
I'd call this `policy` instead of `restrictions`, e.g. `company-a-policy` (here and in the example below)

Also, can you get rid of all the `[s]` in the tree below? These don't have to be subnamespaces (e.g. namespaces with `SubnamespaceAnchor` objects), they just have to be _child_ namespaces.